crates/polars-utils/src/idx_vec.rs

```diff
@@ -295,11 +303,20 @@ macro_rules! unitvec {
     );
     ($elem:expr) => (
         {let mut new = $crate::idx_vec::UnitVec::new();
+        let v = $elem;
         // SAFETY: first element always fits.
-        unsafe { new.push_unchecked($elem) };
+        unsafe { new.push_unchecked(v) };
         new}
     );
     ($($x:expr),+ $(,)?) => (
             vec![$($x),+].into()
     );
 }
+
+mod tests {
+    #[test]
+    fn test_unitvec() {
+        let v = unitvec![1usize];
```

Comment:
Can you change this to use a constant different than 1? Might give false-negatives with such a common constant.

Reply:
Added a list of values